class-tgm-plugin-activation.php

```diff
@@ -2095,7 +2095,7 @@ protected function _gather_plugin_data() {
 				if ( ! empty( $upgrade_notice ) ) {
 					$table_data[ $i ]['upgrade_notice'] = $upgrade_notice;
 
-					add_action( "tgmpa_after_plugin_row_$slug", array( $this, 'wp_plugin_update_row' ), 10, 2 );
+					add_action( 'tgmpa_after_plugin_row_' . $slug, array( $this, 'wp_plugin_update_row' ), 10, 2 );
```

Comment:
This bit needs reverting. Interpolation should be used for hooks, so that WP-Parser can correctly pick up the name for documenting the hooks. We can probably turn this check off for Scrutinizer if necessary.

https://core.trac.wordpress.org/ticket/26869#comment:29